tests/spec/fs.mkdtemp.spec.js

```diff
@@ -0,0 +1,50 @@
+var util = require('../lib/test-utils.js');
+var expect = require('chai').expect;
+
+describe('fs.mkdtemp', function() {
+  beforeEach(util.setup);
+  afterEach(util.cleanup);
+
+  it('should be a function', function() {
+    var fs = util.fs();
+    expect(fs.mkdtemp).to.be.a('function');
+  });
+
+  it('should craete temp dir with specified prefix', function(done) {
+    var fs = util.fs();
+    fs.mkdtemp('/foo', function(error, path) {
+      expect(error).not.to.exist;
+      expect(path).to.match(/foo.{6}/);
```

Comment:
I think you want to use `/^\w{7}$/` instead, or perhaps ``/^\-\w{6}$/`.  Same for the others in this file.

Reply:
This got missed throughout this file, can you update your regex?  It's working now, but only testing the first 6 characters of 7, and using `.` is too much I think.